packages/amplify-graphql-transformer-core/src/transformer-context/resolver.ts

```diff
@@ -265,8 +265,9 @@ export class TransformerResolver implements TransformerResolverProvider {
     );
     if (authModes.includes(AuthorizationType.IAM)) {
       const authRoleParameter = (context.stackManager.getParameter(IAM_AUTH_ROLE_PARAMETER) as CfnParameter).valueAsString;
-      const unauthRoleParameter = (context.stackManager.getParameter(IAM_UNAUTH_ROLE_PARAMETER) as CfnParameter).valueAsString;
-      initResolver += dedent`\n
+			const unauthRoleParameter = (context.stackManager.getParameter(IAM_UNAUTH_ROLE_PARAMETER) as CfnParameter).valueAsString;
+			initResolver += '\n'
```

Comment:
It looks like you're moving the line break and not removing - can you explain why this is needed? Thank you.

Reply:
@danielleadams Sure. `ts-dedent wasn't inserting a line break, it was inserting `\n` as plain text. There are a bunch of [issues](https://github.com/dmnd/dedent/issues?q=is%3Aissue) on dedent around not escaping characters, so perhaps there are some odd quirks around how it parses the string.